lib/propono/services/publisher.rb

```diff
@@ -5,8 +5,14 @@ class PublisherError < ProponoError
   end
 
   class Publisher
-    def self.publish(*args)
-      new(*args).publish
+    if RUBY_VERSION < '3'
+      def self.publish(*args)
+        new(*args).publish
+      end
```

Comment:
I think `...` might actually be better here than `*args`, as that will allow for any kwparams in the future.

Btw, there's also [ruby2_keywords](https://github.com/ruby/ruby2_keywords) which might be useful (grep `ruby2_keywords` [here](https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/) for some more info.

I don't think I have a preference though as long as it works.

Reply:
OK, yeah, I'm happy to leave it as it is :)